rust/operator/src/kafka_controller.rs

```diff
@@ -205,6 +207,25 @@ pub async fn reconcile_kafka(
         )
         .await
         .context(ApplyRoleRoleBindingSnafu)?;
+
+    // Assemble the OPA connection string from the discovery and the given path if provided
+    // Will be passed as --override parameter in the cli in the state ful set
+    let opa_connect = if let Some(opa_spec) = &kafka.spec.opa {
+        Some(
+            opa_spec
+                .full_document_url_from_config_map(
+                    client,
+                    &*kafka,
+                    Some("allow"),
```

Comment:
I guess we decided on hard-coding the rule? ;) (I'm not that into OPA)

Reply:
Yeah the kafka opa authorizer expects every request on allow and sends all metadata via "input". So there is no other option here (e.g. compared to the trino one).

Reply:
Thanks!